README.md

```diff
@@ -141,6 +141,16 @@ $ meson install -C build
 
 Notes: `meson setup` will also clone both libbpf and bpftool repos and `meson compile` will build them both.
 
+### Static linking against system libbpf
+Note, depending on your system configuration `libbpf.a` may be in a different
+directory.
+ ```
+$ cd $SCX
+$ meson setup build --prefix ~ -D libbpf_a=/usr/lib64/libbpf.a
```

Comment:
Can you also add the include override? Depending on the version difference matching custom .a with system headers might not work too well.

Reply:
Good catch! Did some more digging and #259 is probably a better solution compared to using the system libbpf.